ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml

```diff
@@ -1059,8 +1059,8 @@ tests:
     test:
     - ref: openshift-extended-test
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-winc
-- as: aws-ipi-private-f6-disasterrecovery
-  cron: 11 11 1,7,13,19,25 * *
+- as: aws-ipi-private-f28-disasterrecovery
```

Comment:
I saw there are some disasterrecovery jobs in arm ci:
```
$ grep -r "as: .*disasterrecovery" *release-4.15*
...
openshift-openshift-tests-private-release-4.15__arm64-nightly.yaml:- as: aws-ipi-private-f14-disasterrecovery
openshift-openshift-tests-private-release-4.15__arm64-nightly.yaml:- as: azure-ipi-private-f14-disasterrecovery
openshift-openshift-tests-private-release-4.15__arm64-nightly.yaml:- as: gcp-ipi-private-f14-disasterrecovery
```
Shall we remove them in arm yaml or change the frequency to f28, too?

Reply:
I have not enable dr on arm64 before, is it updated automatically? anyway, I agree to remove it from arm.

Reply:
I am not sure how they are shown there, cc @aleskandro do you have any comments on the `disasterrecovery` testing on arm?

Reply:
We should have some jobs there too imho. Not sure why you never created test configs on arm before but we could balance some jobs to improve coverage and also reduce costs as these should take some time to run iirc

Reply:
@aleskandro we may think about update dr in arm, and this pr is for amd only, I think we may not mix these two platform in one pr, is it make sense?

Reply:
@geliu2016 Thanks. 

In the past, we agreed that any test configs regarding a platform, pushed after the enablement of that platform for arm, should have been for both arm64 and amd64, unless specific cases that can have architecture-specific issues (and that multiarch QE should be made aware of). 

I see some disasterrecovery testconfigs for arm64 are already there since the relevant platforms were enabled.

Although you might not have pushed them, the process of cloud platforms onboarding for an architecture about this is:
1. Enablement from multi-arch QE
2. after GA, the responsibility and maintenance lay on the relevant teams responsible for the "features", regardless of the arch.

Otherwise, we should run a continuous 'reconcile loop' after enablement, which doesn't scale: 2 people following the whole team pushing amd64-only testconfigs should stay continuously on Git Hub watching your amd64-only PRs.

As this is a cron update, you can distribute the frequency of runs across architecture. If you wanted frequency to F14 for amd64, you can set F28 on amd and F28 on arm in this PR. 

Other work is being done to move things towards the multi payload by default, but it is out of the scope for this PR.

We should definitely retire the 'pr on amd64 first' approach. Let's help doing it :)

Reply:
@aleskandro could you please take a quick view? thanks